doc/sidecar_configmap.md

```diff
@@ -124,30 +125,44 @@ Description of `chaosfs`:
 * **addr**: defines the address of the grpc server, default value: ":65534".
 * **pidfile**: defines the pid file to record the pid of the `chaosfs` process.
 * **original**: defines the target directory that need to be injected file system IO fault.
-This value should be set to the data directory of the target application.
-* **mountpoint**: defines the mountpoint to mount original directory.
 This directory is usually set to the same level directory as the original directory.
+* **mountpoint**: defines the mountpoint to mount original directory.
```

Comment:
Seems this description is contradictory with how it is used actually?

Reply:
yes, I will fix it.